src/args/arg_builder/flag.rs

```diff
@@ -11,19 +11,25 @@ use vec_map::{self, VecMap};
 
 // Internal
 use Arg;
-use args::{ArgSettings, Base, Switched, AnyArg, DispOrder};
+use args::{AnyArg, ArgSettings, Base, DispOrder, Switched};
```

Comment:
let me know if you don't want the default rustfmt changes to be merged in... this is current rustfmt, at least up to a few days ago.

Reply:
I'm good with the rustfmt changes. I try to stay using the latest and greatest update so as to stick with community best practices.